packages/components/Organisms/CampaignPrimary/index.vue

```diff
@@ -77,23 +84,27 @@ export default {
   $rpl-campaign-primary-summary-text-color: mix(rpl_color('extra_dark_neutral'), rpl_color('white'), 93%) !default;
   $rpl-campaign-primary-summary-margin-xs: ($rpl-space * 5) 0 ($rpl-space * 7) !default;
   $rpl-campaign-primary-summary-margin-s: ($rpl-space * 6) 0 !default;
-  $rpl-campaign-primary-content-padding-xs: 0 $rpl-component-padding-xs !default;
-  $rpl-campaign-primary-content-padding-s: 0 $rpl-component-padding-s !default;
-  $rpl-campaign-primary-button-margin-s: 0 $rpl-component-padding-s !default;
+  $rpl-campaign-primary-caption-ruleset: ('xxs', 1.17em, 'medium') !default;
+  $rpl-campaign-primary-caption-margin-xs: ($rpl-space * 5) 0 0 !default;
+  $rpl-campaign-primary-caption-margin-s: ($rpl-space * 12) 0 0 !default;
```

Comment:
I think as long as we using `$rpl-space`, not really need to create sass variables for each margin and padding. Unless they are reused in several places.

Reply:
As discussed, this should be discussed as a component-wide change.